lib/compat/wordpress-6.0/block-template-utils.php

```diff
@@ -99,10 +99,18 @@ function gutenberg_generate_block_templates_export_file() {
 	// Load theme.json into the zip file.
 	$tree = WP_Theme_JSON_Resolver_Gutenberg::get_theme_data();
 	$tree->merge( WP_Theme_JSON_Resolver_Gutenberg::get_user_data() );
+	$theme_json_raw = $tree->get_data();
+	// If a version is defined, add a schema.
+	if ( $theme_json_raw['version'] && in_array( $theme_json_raw['version'], array( "1", "2" ) ) ) {
```

Comment:
The issue with checking for `1` or `2` is that when the version is upgraded this code needs to as well. Given that worst-case scenario is that the schema is not yet published, can we use what's in `version`: it's going to be always the latest.

Reply:
Hmmm, but https://json.schemastore.org/theme-v2.json doesn't exist?

Reply:
Oh, interesting. I just saw Sarah's comment above https://github.com/WordPress/gutenberg/pull/39775#discussion_r835757763 pointing to https://make.wordpress.org/themes/2021/11/30/theme-json-schema/

According to that, it sounds like we need to use different URL depending on whether we run in the plugin or core. I'll ping @ajlende for confirmation but it seems that we need:

- `https://schemas.wp.org/trunk/theme.json` in the plugin
- `https://schemas.wp.org/wp/<version>/theme.json` in core

We need to do something to remember doing the change when this code is ported to core. Perhaps a comment would sufice.